UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Logging/EventIds.cs

```diff
@@ -716,7 +716,19 @@ public enum EventIds
         /// <summary>
         /// 805177 - Request for creating serial.aio file in exchange set is failed.
         /// </summary>
-        SerialAioFileIsNotCreated = 805177
+        SerialAioFileIsNotCreated = 805177,
+        /// <summary>
+        /// 805178 - Request for creating catalog.031 file in aio exchange set is failed.
+        /// </summary>
+        CatalogFileForAioIsNotCreated = 805178,
+        /// <summary>
+        /// 805179 - Request for creating catalog.031 file in aio exchange set is started.
+        /// </summary>
+        CreateCatalogFileForAioRequestStart = 805179,
+        /// <summary>
+        /// 805180 - Request for creating catalog.031 file in aio exchange set is completed.
+        /// </summary>
+        CreateCatalogFileForAioRequestCompleted = 805180,
```

Comment:
```suggestion
      
```
I think, as we have added temporary method for catalog so eventIds added with new method are also temporary. Please remove if this consumption is correct and use existing eventIds.

Reply:
Resolved